tests/e2e/linalg_ext_ops/BUILD.bazel

```diff
@@ -38,6 +41,21 @@ iree_check_single_backend_test_suite(
     target_backend = "llvm-cpu",
 )
 
+iree_check_single_backend_test_suite(
+    name = "check_no_dt_llvm-cpu_local-task",
+    srcs = [
+        "custom_op.mlir",
+    ],
+    compiler_flags = [
+        "--iree-opt-data-tiling=false",
```

Comment:
Is this separated because data tiling applies to the contents of the custom op? We should [add a TODO to] change that if so.